python/ray/tests/BUILD

```diff
@@ -306,14 +306,6 @@ py_test(
     deps = ["//:ray_lib"],
 )
 
-py_test(
```

Comment:
where are we running job manager test on CI after this move ?

Reply:
it gets run in the dashboard tests (there's a wildcard there that runs all tests in subdir)

Reply:
https://buildkite.com/ray-project/ray-builders-pr/builds/18488#8a6f5be9-5bee-4820-88b4-472a8fccbe51/461-3399